### PR TITLE
Fixed clear search issue #1230

### LINKED ISSUE #1230: Clear search bar input on component load
**I'm submitting a ...**
- [x] bug report
- [ ] feature request

**Current behavior:**

When the brand logo is clicked to go back to homepage, the search input is not cleared. It displays the previous search string

**Expected behavior:**

On loading search bar component, the search input is cleared.

**Screenshots of the issue:**

Results page: 
screenshot from 2018-11-20 01-04-18

After selecting Susper logo:
screenshot from 2018-11-20 01-04-25

### PR DESCRIPTION
**Pull request by reeversedev**
Fixes #1230 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)
- [x] Added Surge preview link

#### Changes proposed in this pull request:

- Clears the input after clicking the logo on navbar.

- https://pr-1343-fossasia-susper.surge.sh

### CODE REVIEW

**Comment by reeversedev** [comment]
@simsausaurabh @mariobehling Please review. Thanks.

File: src/app/search-bar/search-bar.component.ts [hunk 1]
```diff
@@ -10,35 +10,35 @@ import {
   ViewChild,
   ViewChildren,
   ElementRef,
-  HostListener,
-} from '@angular/core';
-import { ActivatedRoute, Router } from '@angular/router';
-import { Store } from '@ngrx/store';
-import * as fromRoot from '../reducers';
```
Comment (simsausaurabh) [inline src/app/search-bar/search-bar.component.ts]: Why changing to double quotes? Its standard used in project.

**Comment by shreyanshdwivedi** [comment]
@reeversedev  I'll suggest you to add proper test for the function you added. We are working to increase the coverage of every component and your PR decreases coverage of `navbar component` by `6%`
If you want, you can open a separate issue to refactor and increase coverage of navbar component.

**Comment by reeversedev** [comment]
@shreyanshdwivedi Alright. I'll create a separate issue for that. Till that can you put this in WIP tag?

**Comment by shreyanshdwivedi** [comment]
@reeversedev we can proceed by merging this if maintainers are okay with your pull request. Afterwards, you can create an issue and work on that and create a separate PR for refactoring and increasing code coverage :)

**Comment by reeversedev** [comment]
@shreyanshdwivedi Yeah. Let's see if mentors are good to go with this.

**Pull request merged**